dist/aphrodite.js

```diff
@@ -218,6 +218,9 @@ module.exports =
 	 */
 	var isUnitlessNumber = {
 	    animationIterationCount: true,
+	    borderImageOutset: true,
```

Comment:
It's because you ran the build and committed the dist directory. This is why I recommend not committing generated files: “Why I don’t commit generated files to master” @kentcdodds https://medium.com/@kentcdodds/why-i-don-t-commit-generated-files-to-master-a4d76382564